src/components/ViewRating.tsx

```diff
@@ -15,7 +15,7 @@ const ViewRating = ({ review }: { review: ProductReview }) => {
         </p>
       </div>
       <div className="w-full">
-        <p>{formatDateTime(new Date(review.added_at))} </p>
+        <time>{formatDateTime(new Date(review.added_at))} </time>
```

Comment:
Please check if the value is valid. Otherwise you'll need to use the `dateTime` attribute.